blkmaker.c

```diff
@@ -322,21 +322,21 @@ bool _blkmk_calculate_witness_mrklroot(blktemplate_t * const tmpl, libblkmaker_h
 		if (txn->txid && memcmp(txn->hash_, txn->txid, sizeof(*txn->txid))) {
 			*witness_needed = true;
 		}
-		memcpy(&hashes[i + 1], txn->hash_, sizeof(*hashes));
+		memcpy(&hashes[i + 1], txn->hash_, sizeof(libblkmaker_hash_t));
```

Comment:
Understood, but I wasn't sure if that caused *hashes to be evaluated at runtime or not. I assume not, but since it's undefined without c99, i was playing it safe.

I guess you'd like these reverted?